addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -220,6 +220,7 @@ export class OdooEditor extends EventTarget {
                 toSanitize: true,
                 isRootEditable: true,
                 placeholder: false,
+                pointerInsideCell: false,
```

Comment:
Options are not meant to hold a state. If you absolutely need something like this, use a separate instance variable. But you don't need it anyway because your code can be contained in `OdooEditor`.